saw-core/prelude/Prelude.sawcore

```diff
@@ -1401,53 +1767,210 @@ composeM : (a b c: sort 0) -> (a -> CompM b) -> (b -> CompM c) -> a -> CompM c;
 composeM a b c f g x = bindM b c (f x) g;
 
 -- Raise an error in the computation monad
-primitive errorM : (a:sort 0) -> CompM a;
+primitive errorM : (a:sort 0) -> String -> CompM a;
```

Comment:
This and other `CompM`-related changes might make sense as a separate PR, given that we already have `CompM` in the saw-core prelude on the master branch.

Reply:
Sure that makes sense. What actions do you need from me?

On a related note: we are going to be making more changes to the `CompM` monad in the next month or so as part of our shift to using itrees to implement `CompM`. Not only will there be new combinators in the prelude, but there will also be a new dependency on the itrees library in `saw-core-coq`. Should we wait on merging these `CompM` changes until the itrees-related changes are resolved? How should we discuss the plan for those changes?